heat/core/tests/test_statistics.py

```diff
@@ -888,6 +890,67 @@ def test_minimum(self):
         with self.assertRaises(ValueError):
             ht.minimum(random_volume_1, random_volume_2, out=output)
 
+    def test_percentile(self):
```

Comment:
can you add a different dtype to the tests?

Reply:
Done, as in I left the original data tensor as int. ht.percentile as well as numpy.percentile always returns float64.